### PR TITLE
[CLN][PRF] sale: another cleanup pass before v16

### PR DESCRIPTION
**Pull request by Feyensv**
Enterprise PR: https://github.com/odoo/enterprise/pull/30408

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/sale/models/sale_order.py [hunk 1]
```diff
@@ -822,7 +841,7 @@ def action_done(self):
             if tx and tx.state == 'pending' and tx.acquirer_id.provider == 'transfer':
                 tx._set_done()
                 tx.write({'is_post_processed': True})
-        return self.write({'state': 'done'})
+        self.write({'state': 'done'})
```
Comment (Demesmaeker) [inline addons/sale/models/sale_order.py:844]: Here for your line =P

**Review by Demesmaeker — APPROVED** [review]
:duck:

**Comment by tde-banana-odoo** [comment]
According to last guidelines, tags should be more like [LINT][PERF] :D 

Tde, doing useful reviews since 2011

**Comment by Feyensv** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**